tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,59 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
```

Comment:
What is the meaning of "outbound" and "inbound" in this function context? Maybe "from" & "to" is more appropriate?